src/bin/pgrx_embed.rs

```diff
@@ -1,2 +1,3 @@
 // https://github.com/pgcentralfoundation/pgrx/issues/1888
+// #![allow(unexpected_cfgs)]
```

Comment:
Not really best practice but felt OK since tests are passing and I can't see any updated info on handling this differently at the github issue https://github.com/pgcentralfoundation/pgrx/issues/1888.

Reply:
I have vague recollections of getting bitten by this a while ago; my browser history suggests https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html may contain useful info (I think I fixed it with some `println!` magicks in `build.rs` or something...)

Reply:
Ah, the CI fail also suggests consulting that page.  Clearly that's where I must have got it from the first time...